Dockerfile

```diff
@@ -3,3 +3,5 @@ FROM nginx:latest
 LABEL org.opencontainers.image.authors="pmc@voxpupuli.org"
 
 COPY . /usr/share/nginx/html/
+
+ENTRYPOINT [ "/usr/share/nginx/html/entrypoint.sh" ]
```

Comment:
It's a bit odd to have the entry point in the document root, but it looks like that's already a preexisting problem.

Reply:
i can copy this to another location...